### PR TITLE
Feature/vcm 7571

### PR DESCRIPTION
**Pull request by msw-ursaspace**
### What's Changed:

- Analytic Descriptions
  - All analytic tool tip descriptions have been updated to provide more detail to users as well as any imagery restrictions
  - All change detection sub-options have been given tool tip descriptions as well 
  - Screen shots of the new descriptions have been added at the bottom of this description
- Updated Error Messaging
  - Error messages for Required AOI Field has been updated to provide more detail and information 

### To Test:
Analytic Descriptions: 
- To view/test that these have been updated hover over the "Information" icon that appears next to the "Analytic Options" and "Change Detection Options" selection pane. (Note for the Change Detection Options to show "Change Detection" must be selected in the "Analytic Options")

Updated AOI Error Message:
To Trigger the AOI Required Error Message do the following:
1. When filling out the “Analytics” form select the pencil to add a new AOI (select point for now). 
1.a This generates a feature layer to add points to
2. This will pre-populate the AOI field with a reference to the feature layer but no points have been added to the feature layer yet 
image
2.a The contents pane will also show that the feature layer (Analytics Area of Interest (Points) 2) has a “Point” on it even tho we haven’t added on yet
image
2.b This causes the analytic AOI field to display the error message since the feature layer is missing a point
 image
2.c Once a point has been added to the map the AOI field error will not clear until the Form is forced to refresh (either by selecting another analytic or updating a different field)

Analytic Desctiptions
Small Vessel Detection:
image
Large Vessel Detection:
image
Aircraft Detection:
image
Ground Vehicle Detection:
image
Change Detection:
image
Ground Displacement:
image
Object Detection (SPEX):
image

Change Detection - Small Objects
image

Change Detection - Medium Structures & Large Areas
image

### CODE REVIEW

File: src/parameters/analytic_options.py [hunk 1]
```diff
@@ -7,21 +7,16 @@
...
 cd_map: dict[str, str] = {
-    "Basic Change Detection": "CHANGE_DETECTION",
-    "Object Level Change Detection": "OBJECT_LEVEL",
-    "Port Traffic": "PORT_TRAFFIC",
-    "Parking Lot": "PARKING_LOT",
-    "Construction": "CONSTRUCTION",
-    "Hurricane Damage": "HURRICANE_DAMAGE",
-    "Large Earth": "LARGE_EARTH",
+    "Small Objects": "SMALL_OBJECTS",
+    "Medium Structures": "Medium Structures",
```
Comment (ursaspace) [inline src/parameters/analytic_options.py:18]: for consistency, should be change this to MEDIUM_STRUCTURES ? or is there a functionality reason why it should be left like this?

**Comment by ursaspace** [comment]
It might make sense visually to indent the mode and resolution restrictions (so that they are one tab in from the `Imagery Restrictions:`) so that it better indicates those are a part of the imagery restrictions category. Feel free to play around with this and disagree with me though

**Comment by ursaspace** [comment]
For the change detection options, I think we can remove the results sentence from each of those. It feels a little redundant to list it each time when it's the same exact sentence, plus we already state the same information in the change detection analytic tooltip

**Comment by ursaspace** [comment]
Screenshot 2023-11-03 at 11 52 22 AM

there's a typo in the last sentence -> _Please see the **individual**_

**Comment by ursaspace** [comment]
Screenshot 2023-11-03 at 11 50 29 AM

the phrase 'recognizing large vessels larger than 150 meters' feels redundant. I'd maybe take out that first large.

Can also add large in the first sentence so it says 'The large vessel detection analytic' to match our small detection description

**Comment by ursaspace** [comment]
Screenshot 2023-11-03 at 11 50 03 AM

I'd add an 'is' between 'algorithm' and 'responsible'

File: src/parameters/analytic_options.py (hunk 1 above)
Comment (msw-ursaspace) [reply src/parameters/analytic_options.py:18]: great catch, this was a typo!

**Review by ursaspace — APPROVED** [review]

**Pull request merged**